UpsertedUserDataProcessingOrchestrator/handler.ts

```diff
@@ -46,7 +45,7 @@ export const handler = function*(
     errorOrUpsertedUserDataProcessingOrchestratorInput.value;
 
   yield context.df.callActivity("SendUserDataProcessingEmailActivity", {
-    upsertedUserDataProcessingOrchestratorInput
+    ...upsertedUserDataProcessingOrchestratorInput
```

Comment:
I have to reintroduce this due to ActivityInput compatibility. It expects a Json with this structure: `{"choice": "xxxx",
          "fiscalCode": "xxxxx"}`, while removing `...` we got `{
       "upsertedUserDataProcessingOrchestratorInput": {
          "choice": "XXXXX",
          "fiscalCode": "XXXX",
       },
      }`

Reply:
not if you get rid of the brackets as well. 

`f({ ...obj })` equals `f(obj)`  there's no need to spread the object